nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -91,6 +91,10 @@
 
 ## Backward Incompatibilities {#sec-release-23.11-incompatibilities}
 
+- `network-online.target` is no longer required by `multi-user.target`. This improves overall boot times and is consistent with the behaviour of other distributions.
+
+  Relatedly, `network-online.target` has been fixed to no longer time out for systems with `networking.useDHCP = true` and `systemd.network.enable = true`.
```

Comment:
Should probably say `networking.useNetworkd` instead of `systemd.network.enable`, since the generic network is only configured when the former is enabled.